pkgs/tools/networking/curl/default.nix

```diff
@@ -119,8 +135,8 @@ stdenv.mkDerivation rec {
   meta = with stdenv.lib; {
     description = "A command line tool for transferring files with URL syntax";
     homepage    = https://curl.haxx.se/;
-    maintainers = with maintainers; [ lovek323 ];
```

Comment:
?

Reply:
Right, it's coming from f1693d6cd5c578bb4ac33f344cf8404126ba14f6. Why not.

Reply:
Yep, came from the cherry-pick.